contracts/SBCDepositContract.sol

```diff
@@ -256,6 +256,7 @@ contract SBCDepositContract is
     }
 
     /**
+     * @custom:deprecated
```

Comment:
We cannot deprecate this function without breaking chiado. Client teams should be aware of this and must support this signature until we have a plan for chiado

Reply:
So what change are you suggesting? For me "deprecated" just means that this should no longer be used in the future, but blockchain being blockchain, this is still the signature that needs to be called before some still undefined block on Chiado.

Reply:
deprecated does not convey enough meaning and can lead to confusion. Being this so critical it must be explicitly said that:
- chiado requires this signature from block N to block M, supports both after block M
- gnosis supports both after P